ci/download_from_s3.sh

```diff
@@ -78,10 +84,11 @@ download_lib_docs() {
     "nightly": { "version": .nightly.version, "ucxx_version": .nightly.ucxx_version }
   }' _data/releases.json)
 
+  echo "--- processing active RAPIDS libraries ---"
   PROJECT_MAP=$(yq '.apis + .libs' _data/docs.yml)
 
-  for VERSION_NAME in $(jq -r 'keys | .[]' <<< "$VERSION_MAP"); do
-    for PROJECT in $(yq -r 'keys | .[]' <<< "$PROJECT_MAP"); do
+  for PROJECT in $(yq -r 'keys | .[]' <<< "$PROJECT_MAP"); do
+    for VERSION_NAME in $(jq -r 'keys | .[]' <<< "$VERSION_MAP"); do
```

Comment:
Switching the order of these so that in the logs, all the updates are grouped by library. I personally think it's more common to look at these logs with a question like "which files were copied for `cudf`?" than "which files were copied for all of the nightlies?".

Happy to switch this back if reviewers disagree, I don't feel strongly.